recOrder/pipelines/qlipp_pipeline.py

```diff
@@ -54,10 +54,11 @@ def __init__(self, config: ConfigReader, data: MicromanagerReader, save_dir: str
         self.calib_meta = json.load(open(self.config.calibration_metadata)) \
             if self.config.calibration_metadata else None
         #todo: re-structure reading calib metadata when finalized
-        self.calib_scheme = self.calib_meta['Summary']['~ Acquired Using'] if self.calib_meta \
-            else '4-Frame Extinction'
+        self.calib_scheme = self.calib_meta['Summary']['Acquired Using'] if self.calib_meta \
+            else '4-Frame'
```

Comment:
I'm getting all `pre_post_processing` and `qlipp_pipeline` tests to fail at this line with:
```
E       KeyError: 'Acquired Using'

../recOrder/pipelines/qlipp_pipeline.py:57: KeyError
```

Reply:
Yeah unfortunately the test dataset has a typo in the calibration_metadata file, I am going to create a new test dataset that has that fixed

Reply:
Fixed.